src/glyphs.js

```diff
@@ -142,14 +142,23 @@ const updateMessages = (id, path, req) => {
 */
 const viewPath = "Intro"
 
-const importMessages = (line) => {
+const importMessages = (line, metaTypes) => {
   const message = line.split(",");
   const viewMessage = (message[1] === viewPath);
 
   const newMessage = {"text": message[2], "probability": (viewMessage) ? Number(message[3]):message.splice(3, 5).map(num => Number(num))};
-
+  if (metaTypes && metaTypes.length) newMessage = {...newMessage, "metadata" : []};           //Make a field in the message object for metadata only if metadata exists
   const path = (viewMessage) ? "view" : `view.gauges.${message[1]}`;
 
+  const metaText = (viewMessage) ? message.splice(0,3):message.splice(0,5);
+
+  const metaArray = {}
+  for (let i=0; i<metadata.length; i++) {
+    metaArray.push({`${metaTypes[i]}` : `${metaText[i]}`});
```

Comment:
Since `metaArray` is an object, you can write this as `metaArray[metaTypes[i]] = metaText[i]`